components/support/nimbus-fml/src/main.rs

```diff
@@ -686,4 +697,18 @@ mod cli_tests {
         }
         Ok(())
     }
+
+    ///////////////////////////////////////////////////////////////////////////
+    #[test]
+    fn test_cli_generate_validate() -> Result<()> {
+        let cwd = package_dir()?;
+        let cmd = get_command_from_cli([FML_BIN, "validate", TEST_FILE], &cwd)?;
+
+        assert!(matches!(cmd, CliCmd::Validate(_)));
+
+        if let CliCmd::Validate(cmd) = cmd {
+            assert!(cmd.manifest.ends_with(TEST_FILE));
+        }
```

Comment:
I would write this as

```rust
assert!(matches!(cmd, CliCmd::Validate(c) if c.manifest.ends_with(TEST_FILE)));
```

but either is fine.

Reply:
😮 that's cool